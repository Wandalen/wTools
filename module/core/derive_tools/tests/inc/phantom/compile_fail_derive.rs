use the_module::PhantomData;

#[ derive( PhantomData ) ]
struct MyStruct;

#[ derive( PhantomData ) ]
enum MyEnum
{
  Variant1,
  Variant2,
}

#[ derive( PhantomData ) ]
union MyUnion
{
  field1 : u32,
  field2 : f32,
}