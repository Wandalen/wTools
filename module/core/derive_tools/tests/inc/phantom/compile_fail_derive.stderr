error[E0432]: unresolved import `the_module`
 --> tests/inc/phantom/compile_fail_derive.rs:1:5
  |
1 | use the_module::PhantomData;
  |     ^^^^^^^^^^ use of unresolved module or unlinked crate `the_module`
  |
  = help: if you wanted to use a crate named `the_module`, use `cargo add the_module` to add it to your `Cargo.toml`

error[E0601]: `main` function not found in crate `$CRATE`
  --> tests/inc/phantom/compile_fail_derive.rs:18:2
   |
18 | }
   |  ^ consider adding a `main` function to `$DIR/tests/inc/phantom/compile_fail_derive.rs`
